src/Reflection/ReflectionFunction.php

```diff
@@ -21,6 +22,15 @@ public static function createFromName($functionName)
     }
 
     /**
+     * @param \Closure $closure
+     * @return ReflectionFunction
+     */
+    public static function createFromClosure(\Closure $closure)
+    {
+        return (new FunctionReflector(new ClosureSourceLocator($closure)))->reflect('{closure}');
```

Comment:
Does the fact that a closure source locator is used limit reflection of type hints?

As in:
`$c = function (Foo $foo) : Bar {}`

Can I get reflection for `$foo` and for the return parameter even if a `ClosureSourceLocator` was used?